<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>host修改器 - 我是指针*</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>app_image.ico</normaloff>app_image.ico</iconset>
  </property>
  <widget class="QLineEdit" name="ip_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_format">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存模板</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_format">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_now">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>371</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>模板</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>最终</string>
   </property>
  </widget>
  <widget class="QPushButton" name="now">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>模板到最终</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_host">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>移动到host</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
